ci-operator/config/maistra/istio-images-centos/maistra-istio-images-centos-maistra-2.4.yaml

```diff
@@ -29,6 +29,64 @@ tests:
     make lint
   container:
     from: src-test
+- as: daily-images-2-4
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: openshift-ci
+    product: ocp
+    timeout: 4h0m0s
+    version: "4.12"
+  cron: 00 03 * * *
+  steps:
+    env:
+      MAISTRA_BUILDER_IMAGE: registry.ci.openshift.org/ci/maistra-builder:2.4
+      MAISTRA_MC_POD: not-used
+      MAISTRA_NAMESPACE: maistra-daily-images
+      MAISTRA_SC_POD: maistra-daily-images-pod
+    test:
+    - as: copy-src
+      cli: latest
+      commands: |
+        # copy the script to privileged pod
+        oc cp create-images.sh "${MAISTRA_NAMESPACE}"/"${MAISTRA_SC_POD}":/work/
```

Comment:
copy everything. Not only what's used today. Copying everything will make sure the job works even if the devs change the script to make use of any other file.

Reply:
Question is how to copy everything what we want from pulled repo and do not copy files specific to the pod e.g. .ssh/ (if it exists)